exercises/F-predicates/exercise.js

```diff
@@ -6,11 +6,15 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
-
+function isBetweenZeroAnd10(number) {
+  return number >= 0 && number <= 10;
```

Comment:
Here as well
function isBetweenZeroAnd10(number){
  if (number === 5){
    return true;
}   else {
      return false;
}
}

Reply:
Hi, thanks. Yes. same here. If I create function in this way it will be running correctly for 5, but I want it be true for all numbers "BetweenZeroAnd10".